lib/luhn_cases.rb

```diff
@@ -0,0 +1,27 @@
+require 'exercise_cases'
+
+class LuhnCase < OpenStruct
+  def name
+    'test_%s' % description.tr('- ', '__')
+  end
+
+  def work_load
```

Comment:
I've updated the documentation to help clarify this: https://github.com/exercism/xruby/pull/560